src/include/migraphx/op/broadcast.hpp

```diff
@@ -34,31 +34,29 @@ struct broadcast
         return pack(f(self.axis, "axis"));
     }
 
-    shape broadcast_shape;
+    std::vector<std::size_t> broadcast_lens;
```

Comment:
We should probably add this field to the reflect method.

Reply:
What field should we add? keeping the current implementation?

Reply:
Fixed